roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -421,7 +421,7 @@
   # Keep this aligned with:
   # https://github.com/oVirt/ovirt-engine/blob/master/packaging/playbooks/roles/ovirt-provider-ovn-driver/tasks/main.yml
   - name: Reconfigure OVN central address
-    ansible.builtin.command: vdsm-tool ovn-config {{ engine_vm_ip.stdout_lines[0] }} {{ he_mgmt_network }}
+    ansible.builtin.command: vdsm-tool ovn-config {{ engine_vm_ip.stdout_lines[0] }} {{ he_mgmt_network }} {{ he_fqdn }}
```

Comment:
`he_fqdn` is the engine FQDN, for the host FQDN you can use `he_host_name`.
The variables are described here: https://github.com/oVirt/ovirt-ansible-collection/tree/master/roles/hosted_engine_setup#general-variables

Reply:
done